Makefile

```diff
@@ -123,6 +123,10 @@ wallet_backend_dep: dep
 
 # Cross Compilation Targets (xgo)
 
+.PHONY: repository_index
+repository_index:
+	@aws s3 ls releases.kowala.io | cut -b32- - > index.txt
```

Comment:
That's clever. It's a shame the aws command doesn't support structured output, but it will do.

Reply:
just a temp helper to generate the file, will sort out later on CI